xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/user/impl/xwiki/XWikiGroupServiceImpl.java

```diff
@@ -195,19 +195,9 @@ public void flushCache()
         }
     }
 
-    /**
-     * Check whether the configuration specifies that every user is implicitly in XWikiAllGroup. Configured by the
-     * {@code xwiki.authentication.group.allgroupimplicit} parameter in {@code xwiki.cfg}.
-     *
-     * @param context the current XWiki context
-     * @return {@code true} if the group is implicit and all users should be by default in it, {@code false} if the
-     *         group behaves as all other groups, containing only the users/subgroups that are explicitly listed inside
-     *         the document.
-     */
-    protected boolean isAllGroupImplicit(XWikiContext context)
+    private boolean isAllGroupImplicit(XWikiContext context)
```

Comment:
You should not change the visibility. Since XWikiGroupServiceImpl is a public class you what do do here is the same as removing a method from an interface. If you build with -Pquality you should get an error.